src/main/scala/mesosphere/marathon/api/akkahttp/v2/GroupsController.scala

```diff
@@ -20,8 +29,19 @@ class GroupsController(electionService: ElectionService, infoService: GroupInfoS
     val authorizer: Authorizer
 ) extends Controller {
   import Directives._
+  import mesosphere.marathon.api.akkahttp.EntityMarshallers._
+  import mesosphere.marathon.api.v2.json.Formats._
 
-  def groupDetail(groupId: PathId): Route = ???
+  def groupDetail(groupId: PathId)(implicit identity: Identity): Route = {
+    extractEmbeds {
+      case (appEmbed, groupEmbed) =>
+        onSuccess(infoService.selectGroup(groupId, authorizationSelectors, appEmbed, groupEmbed)) {
+          case Some(info) => complete(Json.toJson(info))
```

Comment:
Hm, shouldn't there be a RAML for `info`?

Reply:
well there is not... :( I was hoping it was one of those types @meln1k was preparing because he is not using RAML type for this in AppsController too.

So what should we do? Keep it as it is? One of us should prepare the type? (which is potentially huge...)

Reply:
Ah, this is the AppInfo, right? Ok, let's just add a comment and a JIRA.

Reply:
no this is groupInfo - but AppInfo is part of GroupInfo - ticket here https://jira.mesosphere.com/browse/MARATHON-7900

Once AppInfo is done, GroupInfo will be very easy...